examples/hoa-renderer.html

```diff
@@ -0,0 +1,171 @@
+<!--
+  Copyright 2017 Google Inc. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Example: HOARotator</title>
+  <script src="audiocontext-shim.js"></script>
+  <script src="../build/omnitone.js"></script>
+  <link rel="stylesheet" href="example.css">
+  <style>
+    .slider-width500
```

Comment:
just for style, otherwise the slider is very small & it's difficult to get accurate angles. it can be removed though.